### ISSUE TITLE
ChromeDriver Executable - Smaller?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [FuckTheWorld/chromedriver#128]
```
What steps will reproduce the problem?
1. Go to the downloads page. 
http://code.google.com/p/chromedriver/downloads/list
2. look at the size decrease for the windows executable, from the may 23 to the 
july 10 versions 

What Happens.

Well using the win_20 version my tests run good, everything behaves as it 
should. but when i switched out my chromedriver for the new one (win_22) the 
tests couldn't even progress past the second page.

so i switched back to the reliable win_20 chromedriver. 

But I am curious as to how the size of the chromed driver could decrease so 
dramatically from 14.9 to 2.99 (uncompressed) to me it feels like functionality 
would have been lost if the size got that much smaller, and my automation test, 
would seem to indicate the same (at least for me). leading me to conclude that 
the smaller driver might be missing features.


What version of the product are you using? On what operating system?
ChromeDriver = win_20 and win_22
Chrome = 20.0.1132
OS = XP

Please attach chromedriver.log to your bug report if possible.
log file only indicates that the element couldn't be found because chromedriver 
got ahead of itself submitting things before it actually clicked the submit 
button and other oddities similar to that.


Basically I'm curious as to why the new windows chrome driver is so much 
smaller than the old windows chrome driver.
```

Original issue reported on code.google.com by `williama...@gmail.com` on 1 Aug 2012 at 3:10
- Merged into: #121
